DirectProgramming/C++SYCL_FPGA/Tutorials/Features/ip_authoring_interfaces_overview/sample.json

```diff
@@ -0,0 +1,71 @@
+{
+    "guid": "7d8482f5-39f1-4cf1-aa2e-a1f72cfc47cb",
+    "name": "IP Authoring Interfaces Overview",
+    "categories": ["Toolkit/oneAPI Direct Programming/C++SYCL FPGA/Tutorials/Features"],
+    "description": "Intel® FPGA example designs oversewing different interfaces for IP Authoring",
+    "toolchain": ["icpx"],
+    "os": ["linux", "windows"],
+    "targetDevice": ["FPGA"],
+    "builder": ["ide", "cmake"],
+    "languages": [{"cpp":{}}],
+    "commonFolder": {
+      "base": "../../..",
+      "include": [
+        "README.md",
+        "Tutorials/Features/ip_authoring_interfaces_overview",
+        "include"
+      ],
+      "exclude": []
+    },
+    "ciTests": {
+      "linux": [
+        {
+          "id": "fpga_emu",
+          "steps": [
+            "icpx --version",
+            "mkdir build",
+            "cd build",
+            "cmake ..",
+            "make fpga_emu",
+            "./vector_add.fpga_emu"
+          ]
```

Comment:
refering to fpga_compile json, fpga_compile has each subfolder to have another set of citests.
I am not pretty sure how json file should work here but at least i wanted to point out the difference I notice.
<img width="280" alt="image" src="https://github.com/KevinUTAT/oneAPI-samples_wenkaixu_fork/assets/114061659/2b46d6d9-3e0f-40ad-b55b-bb9f431c609b">

Reply:
I am not sure how the CI steps works, but i will try mimic the one from `fpga_compile`

Reply:
To answer the suggestion above:
I will change the wording around the `fpga` compilation target. We still nee this target to estimate aera with quartus.

Reply:
> I am not sure how the CI steps works, but i will try mimic the one from `fpga_compile`

Did you skip "make report" and  "nmake report" for csr-pipes in purpose? I see that you have added both make fpga_emu and report for others

Reply:
The CI works by executing the commands listed in "steps" one by one
So the sample.json should contain targets for fpga_emu and report and the CI machine does not have modelsim and quartus installed